app/controllers/application_controller.rb

```diff
@@ -1341,7 +1343,13 @@ def check_privileges
     end
 
     pass = %w(button x_button).include?(action_name) ? handle_button_rbac : handle_generic_rbac
-    $audit_log.failure("Username [#{current_userid}], Role ID [#{current_user.miq_user_role.try(:id)}] attempted to access area [#{controller_name}], type [Action], task [#{action_name}]") unless pass
+    unless pass
+      $audit_log.failure(_("Username [%{user_id}], Role ID [%{role_id}] attempted to access area [%{name}], type [Action], task [%{action_name}]") %
```

Comment:
Actually, since this is `$audit_log.failure()`, i.e. the message will be put into a log file, the message
should not be translated and needs to stay in english (i.e. the gettext needs to go away).